### PR TITLE
Update provisioning script

### PR DESCRIPTION
**Pull request by SEAjamieD**
The provisioning script no longer lives in `src/` - updated to reflect

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Before you can successfully build, you need to provision the OS with some basic
 
 1. Use `visudo` to grant your account the ability to do passwordless sudo. Add a line similar to the following to the end of your sudoers file: `<username> ALL=(ALL) NOPASSWD: ALL`
 
-2. Run the following provisioning script: `/src/support/linux/provision.sh`
+2. Run the following provisioning script: `/support/linux/provision.sh`
```
Comment (chefsalim) [inline DEVELOPING.md]: This doesn't seem correct as "/support" is not a root level dir?

Comment (SEAjamieD) [reply DEVELOPING.md]: maybe I was running it from in the wrong folder?  I was in `builder` and from there i don't see any sort of `src` : https://github.com/habitat-sh/builder

Comment (chefsalim) [reply DEVELOPING.md]: I think this step should just be:

2. Run the following provisioning script from the root of your Builder repo: 
`
./support/linux/provision.sh
`

Comment (SEAjamieD) [reply DEVELOPING.md]: sounds good to me! i'll update

Comment (chefsalim) [reply DEVELOPING.md]: Reminder to update so we can close this PR out

Comment (SEAjamieD) [reply DEVELOPING.md]: Thanks for the reminder!  Updated and ready to merge.

**Pull request merged**